### PR TITLE
Use container image from quay

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by frenzymadness** [comment]
What is the motivation for this change? Any differences between the container images?

**Comment by fridex** [comment]
> What is the motivation for this change? Any differences between the container images?

None just pushed to our "thoth-station" organization and out of dockerhub.

**Comment by frenzymadness** [comment]
> > What is the motivation for this change? Any differences between the container images?
> 
> None just pushed to our "thoth-station" organization and out of dockerhub.

And what will happen when I update the Dockefile on Github and release a new version? Will it rebuild there as well?

**Comment by fridex** [comment]
> > > What is the motivation for this change? Any differences between the container images?
> > 
> > 
> > None just pushed to our "thoth-station" organization and out of dockerhub.
> 
> And what will happen when I update the Dockefile on Github and release a new version? Will it rebuild there as well?

Nope, it has been pushed manually. The primary use case for this change was to move the image to quay. It can be left as is though or I can add you on quay.

Is there a Dockerfile we should include to this repo to keep it in one place? The image can be automatically build on quay. I don''t know how the original image was produced.

**Comment by frenzymadness** [comment]
The origin of fedora-python-tox is: https://github.com/fedora-python/fedora-python-tox

I am not against moving it to quay but it would be nice to set it in a way that when I update the image on Github it will trigger a rebuild there as if does at docker hub.

File: .travis.yml [hunk 1]
```diff
@@ -5,11 +5,11 @@ services:
   - docker
 
 before_install:
-  - docker pull frenzymadness/fedora-python-tox
+  - docker pull quay.io/thoth-station/fedora-python-tox
```
Comment (goern) [inline .travis.yml:8]: @fridex @harshad16 is there a pipeline to build this?

**Comment by fridex** [comment]
> I am not against moving it to quay but it would be nice to set it in a way that when I update the image on Github it will trigger a rebuild there as if does at docker hub.

Is it some "official fedora tox"?

+1 for automating, I didn't know it's coming from fedora-python org. Is it pushed to your personal account on github once built?

File: .travis.yml (hunk 1 above)
Comment (fridex) [reply .travis.yml:8]: Maybe we will not need any pipeline. Let's see following the discussion in the PR.

**Comment by frenzymadness** [comment]
> Is it some "official fedora tox"?

I'd say yes. It started as my pet project when I was learning about containers and then it became useful so we transferred it to fedora-python org. But fedora-python has no account on Docker hub so it remains in mine.


> +1 for automating, I didn't know it's coming from fedora-python org. Is it pushed to your personal account on ~github~ once built?
> 
> EDIT: not github but dockerhub

Yes, the Docker hub is interconnected with Github and rebuilds the image after every change in master branch.

Give me a moment. I'll try to move the Docker hub repository to some official-looking organization.

**Comment by frenzymadness** [comment]
The image is now available also from a new Docker hub organization: https://hub.docker.com/repository/docker/fedorapython/fedora-python-tox

**Comment by fridex** [comment]
> > Is it some "official fedora tox"?
> 
> I'd say yes. It started as my pet project when I was learning about containers and then it became useful so we transferred it to fedora-python org. But fedora-python has no account on Docker hub so it remains in mine.
> 
> > +1 for automating, I didn't know it's coming from fedora-python org. Is it pushed to your personal account on ~github~ once built?
> > EDIT: not github but dockerhub
> 
> Yes, the Docker hub is interconnected with Github and rebuilds the image after every change in master branch.

Nice, I thought it was a one-time thing hence I wanted to move it here... but it does not make sense as it's a standalone component we rely on and we utilize from elsewhere.

> The image is now available also from a new Docker hub organization: https://hub.docker.com/repository/docker/fedorapython/fedora-python-tox

Awesome, I'll close this PR then. :+1:

**Pull request closed without merging**